### PR TITLE
Social networks

### PR DESCRIPTION
**Pull request by Igor-Arena**

### CODE REVIEW

**Comment by Igor-Arena** [comment]
1. Added pages handling

File: pages/dashboardPage.ts [hunk 1]
```diff
@@ -79,4 +87,16 @@ export class DashboardPage {
   async sortItemsByOption(option) {
     await this.sortingDropdownList.selectOption(option);
   }
-}
+
+  async navigateToSocialNetwork(name) {
+    if ("twitter") {
```
Comment (siarheima-papaya) [inline pages/dashboardPage.ts:92]: You didn't use 'name' inside if(). Condition doesn't work

File: tests/ui_tests/socialNetworks.spec.ts [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  await test.step("Login", async () => {
+    await loginPage.goto();
+    await loginPage.login(
+      standardUserCredentials.userName,
+      standardUserCredentials.userPassword
+    );
+  });
+
+  const allSocialNetworks = ["twitter", "facebook", "linkedin"];
+  for (const socialNetwork of allSocialNetworks) {
```
Comment (siarheima-papaya) [inline tests/ui_tests/socialNetworks.spec.ts:28]: for should be before test. Not before step

File: tests/ui_tests/socialNetworks.spec.ts [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  await test.step (`testing with ${socialNetwork}`, async () => {
+    const pagePromise = context.waitForEvent('page')
```
Comment (siarheima-papaya) [inline tests/ui_tests/socialNetworks.spec.ts:30]: Your code is correct. Change move for() before test and fix navigateToSocialNetwork()

File: pages/dashboardPage.ts [hunk 4]
```diff
@@ -23,6 +27,10 @@ export class DashboardPage {
     this.inventoryItemName = page.locator(".inventory_item_name");
     this.inventoryItemsPrice = page.locator(".inventory_item_price");
     this.sortingDropdownList = page.locator(".product_sort_container");
+    this.twitterIcon = page.locator(".social_twitter a");
```
Comment (siarheima-papaya) [inline pages/dashboardPage.ts:30]: Let's create a function where we can get social network by name. It allows to use parametrised locator.
E.g.
```
    async navigateToSocialNetwork(socialNetworkName) {
        await this.page.locator(`li.social_${socialNetworkName}`).click();
    }
```

**Comment by Igor-Arena** [comment]
1. Added assertions is spec.ts file
2. Added "navigateToSocialNetwork" function

File: pages/dashboardPage.ts [hunk 5]
```diff
@@ -23,6 +27,9 @@ export class DashboardPage {
...
```
Comment (siarheima-papaya) [inline pages/dashboardPage.ts:30]: +

**Review by siarheima-papaya — APPROVED** [review]

**Pull request merged**